docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,228 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+describtion: 'This guide is useful for changing or upgrading the LSP6 Key Manager of a Universal Profile.'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade the LSP6 Key Manager
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its [KeyManager](../../standards/universal-profile/lsp6-key-manager) to follow this guide.
+If you don't have a Universal Profile yet, follow our previous guide [**Create a Universal Profile**](../universal-profile/create-profile.md) or look at the [_lsp-factory.js_](../../tools/lsp-factoryjs/deployment/universal-profile.md) docs.
+
+:::
+
+In this guide, we will learn how to upgrade the LSP6 Key Manager of your Universal Profile to the latest version available.
+
+By the end of this guide, you will know how to:
+
+- Deploy a new LSP6 Key Manager with the last updates.
+- Upgrade your Key Manager by changing the owner of your UP from your old to your new Key Manager.
+
+## Setup
+
+```shell
+npm install @lukso/lsp-smart-contracts web3
+```
+
+## Step 1 - Set up the constants and imports
+
+Create a javascript file and add the following imports on the top of the file.
+Create the following constants:
+
+- `privateKey`: private key of a controller address, **MUST** have [**CHANGEOWNER**](../../standards/universal-profile/lsp6-key-manager.md#permissions) permission.
+- `keyManagerAddress`: address of the current LSP6 Key Manager.
+- `universalProfileAddress`: address of your Universal Profile.
+
+```js
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
+import Web3 from 'web3';
+const web3 = new Web3('https://rpc.l14.lukso.network');
+
+const privateKey = '0x...';
+const universalProfileAddress = '0x...';
+const keyManagerAddress = '0x...';
+```
+
+## Step 2 - Initialize the controller account
+
+In order to send any transaction on the blockchain you need an account, in or case that account MUST have [**CHANGEOWNER**](../../standards/universal-profile/lsp6-key-manager.md#permissions) permission on the Universal Profile that will have its LSP6 Key Manager upgraded.
```

Comment:
"an account, in or case that account" ??

Reply:
in our case